### PR TITLE
Implemented basic WASD controls

### PR DESCRIPTION
**Pull request by chahuistle**
Changes

* Implemented basic WASD + Arrows controls to move around the 2D projection and planet.
* X and Y axis sensitivities are configurable (range: [1, 100]) via the Settings menu.
* X and Y axis can be inverted via the Settings menu.
* Introduced settings are serialized/deserialized to be persist across game runs.
* Added "fine tuning" variables to modify keyboard-based movement.

See: #324

### CODE REVIEW

File: Assets/Scripts/2D/GuiManagerScript.cs [hunk 1]
```diff
@@ -324,6 +334,12 @@ public void OnAutoSave()
 
     private int _lastExLogHash = 0;
 
+    private PointerEventData _keyboardDragTracker = new PointerEventData(EventSystem.current)
+    {
+        position = new Vector2(0, 0),
```
Comment (Zematus) [inline Assets/Scripts/2D/GuiManagerScript.cs:339]: you can use position = Vector2.zero instead

File: Assets/Scripts/2D/ModalPanels/SettingsDialogPanelScript.cs [hunk 2]
```diff
@@ -29,17 +37,118 @@ public void RefreshDevButtonText()
...
+    private bool IsKeyboardAxisSensitivityValid(string text, float min = 1f, float max = 100f)
+    {
+        float parsedValue;
+        try
+        {
+            parsedValue = float.Parse(text);
         }
+        catch (FormatException e)
+        {
+            Debug.Log(String.Format("Invalid axis sensitivity entered {0}", text));
```
Comment (Zematus) [inline Assets/Scripts/2D/ModalPanels/SettingsDialogPanelScript.cs:136]: Change these to Debug.Warning

File: Assets/Scripts/WorldEngine/Cultures/Preferences/CellCulturalPreference.cs [hunk 3]
```diff
@@ -16,7 +16,8 @@ public class CellCulturalPreference : CulturalPreference
     private float _newValue;
 
 #if DEBUG
-    private long _lastUpdateDate = Manager.CurrentWorld.CurrentDate;
+    // Null ref when loading from file
```
Comment (Zematus) [inline Assets/Scripts/WorldEngine/Cultures/Preferences/CellCulturalPreference.cs:19]: Create an issue for this instead of leaving it commented

Comment (chahuistle) [reply Assets/Scripts/WorldEngine/Cultures/Preferences/CellCulturalPreference.cs]: see https://github.com/Zematus/Worlds/issues/350

**Review by Zematus — APPROVED** [review]

**Pull request merged**